cmake/catkin_package.cmake

```diff
@@ -374,6 +377,15 @@ function(_catkin_package)
       endif()
     endforeach()
 
+    set(PKG_COMPILE_OPTIONS ${${PROJECT_NAME}_COMPILE_OPTIONS})
+    separate_arguments(PKG_COMPILE_OPTIONS)
+    separate_arguments(PROJECT_DEFINES)
```

Comment:
I wasn't aware of that, this was only a safe guard. If `cmake_parse_arguments` already takes care of it this line can be omitted.